baselines/powerplatform.md

```diff
@@ -2,63 +2,53 @@
 
 Microsoft Power Platform is a group of applications comprised of a
 low-code application development toolkit, business intelligence software,
-a custom chat bot creator, and app connectivity software. The following sections summarizes
-the various Power Platform applications.
+a custom chat bot creator, and app connectivity software.  This Secure Configuration Baseline (SCB) provides specific policies to strengthen Power Platform security.
```

Comment:
Observation: Not that the product documents must be the same.  The last sentence is so slightly different in phrasing - yet, with a very differnt tone. 

AAD's is "This security baseline provides policies to help secure AAD."

Reply:
updated to keep consistency.